MAUI/Popup/layout-customizations.md

```diff
@@ -0,0 +1,451 @@
+---
+layout: post
+title: Layout Customizations in .NET MAUI Popup control | Syncfusion
+description: Learn here all about Layout Customizations support in Syncfusion .NET MAUI Popup (SfPopup) control, its elements and more.
+platform: MAUI
+control: SfPopup
+documentation: ug
+---
+
+# Layout Customizations in MAUI Popup (SfPopup)
+
+## Popup appearance mode for footer
+
+The `SfPopup` supports two types of `SfPopup.AppearanceMode`. By default, the `PopupButtonAppearanceMode.OneButton` is set. You can change the appearance by using the `SfPopup.AppearanceMode` property.
+
+The two different appearance modes in the SfPopup are as follows:
+<table>
+<tr>
+<th>Modes</th>
+<th>Description</th></tr>
+<tr>
+<td>`OneButton`</td>
+<td>Shows the `SfPopup` with one button (accept button) in the footer view. This is the default value.</td></tr>
+<tr>
+<td>`TwoButton`</td>
+<td>Shows the `SfPopup` with two buttons (accept & decline buttons) in the footer view.</td></tr>
+</table>
+
+In the following code example, set the `SfPopup.AppearanceMode` property as `OneButton` which displays only the Accept button in the footer view.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" hl_lines="11" %}
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:sfPopup="clr-namespace:Syncfusion.Maui.Popup;assembly=Syncfusion.Maui.Popup"
+             x:Class="PopupMauiLayout.MainPage">
+     <ContentPage.Content>
+        <StackLayout Padding="20">
+            <Button x:Name="clickToShowPopup" Text="ClickToShowPopup" 
+                    VerticalOptions="Start" HorizontalOptions="Center" 
+                    Clicked="ClickToShowPopup_Clicked" />
+            <sfPopup:SfPopup x:Name="sfPopup" 
+                             AppearanceMode="OneButton" 
+                             ShowFooter="True">
+            </sfPopup:SfPopup>
+        </StackLayout>
+    </ContentPage.Content>
+</ContentPage>
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" hl_lines="10" %}
+using Syncfusion.Maui.Popup;
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        sfPopup.ShowFooter = true;
+        // Setting the AppearanceMode as OneButton
+        sfPopup.AppearanceMode = PopupButtonAppearanceMode.OneButton;
+    }
+
+    private void ClickToShowPopup_Clicked(object sender, EventArgs e)
+    {
+        sfPopup.Show();
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI Popup with accept button](Images/layout-customizations/maui-popup-appearance-mode-one-button.png)
+
+In the following code example, set the `SfPopup.AppearanceMode` property as `TwoButton` which displays both Accept button and Decline button in the footer view.

```

Comment:
In the following code example, set the `SfPopup.AppearanceMode` property as `TwoButton,` which displays both Accept and Decline buttons in the footer view.